docs/Migrations.md

```diff
@@ -0,0 +1,22 @@
+# Diesel Migrations
+
+## Pre-requisites
+
+* `cargo install diesel_cli --no-default-features --features postgres`
+
```

Comment:
Should this be installed automatically in dev environments? Seems ideal to just have it be present rather than have to remember to install it.